.github/workflows/release-deprecate.yaml

```diff
@@ -0,0 +1,22 @@
+name: "Deprecate Old Versions"
+
+on:
+  workflow_dispatch:
```

Comment:
A `dry_run` flag might be nice here, that is defaulted to true to check the output IMO.